### ISSUE TITLE
1.8 - adding form-control class when I don't want it

### ISSUE THREAD

**Issue by njbarrett** [kristijanhusak/laravel-form-builder#271]
Hey there,
I have the following field setup:
```
            ->add('states', 'entity', [
                'rules' => 'required',
                'class' => 'App\State',
                'multiple' => true,
                'expanded' => true,
                'property_key' => 'id',
                'property' => 'abbreviation',
                'label_attr' => [
                    'class' => 'control-label col-xs-12',
                ],
                'choice_options' => [
                    'wrapper' => ['class' => 'col-sm-3'],
                    'attr' => ['class' => 'col-sm-2'],
                    'label_attr' => ['class' => 'col-sm-10'],
                ],
                'wrapper' => [
                    'class' => 'form-group required col-xs-12',
                ],
                'help_block' => [
                    'text' => 'Select all states your business operates in',
                    'tag' => 'p',
                    'attr' => ['class' => 'help-block col-xs-12']
                ],
            ])
```

Since updating to 1.8, it is adding the `form-control` class to the checkbox input elements, and I am directly specifying the class attribute for the choices. How can I tell it *not* to include form-control classes on those checkboxes? It wasn't adding this class prior to the update.

Thanks for an awesome library.

**Comment by kristijanhusak** [comment]
Yeah that's true, it was added in this pr https://github.com/kristijanhusak/laravel-form-builder/pull/220 , but i thought it will only happen if you add class per type in config. Thanks for reporting, i'll check it out.

**Comment by kristijanhusak** [comment]
I fixed it on `dev-master`, but still need to fix few things before i publish patch version.
You can use `dev-master` without any worries, i'm not pushing any invalid code to it.

**Comment by njbarrett** [comment]
Ok thanks!

**Comment by marcoraddatz** [comment]
I didn't have a look into the code. But does this also remove the class from the button?

**Comment by kristijanhusak** [comment]
@marcoraddatz it works same for all field types.

This is now available on version 1.9.

Thanks for reporting.
